lib/ansible/modules/cloud/amazon/ec2.py

```diff
@@ -1193,7 +1194,15 @@ def create_instances(module, ec2, vpc, override_count=None):
                 # (the default) or 'terminate' here.
                 params['instance_initiated_shutdown_behavior'] = instance_initiated_shutdown_behavior or 'stop'
 
-                res = ec2.run_instances(**params)
+                try:
+                    res = ec2.run_instances(**params)
+                except boto.exception.EC2ResponseError as e:
+                    if "InvalidParameterCombination" == e.error_code:
```

Comment:
Can we narrow this check a little bit, since re-running this call will only help if `InstanceInitiatedShutdownBehavior` is set to `stop`? How about `if "InvalidParameterCombination" == e.error_code and params['instance_initiated_shutdown_behavior'] != 'terminate'` so if that's already the case, we just reraise and don't make the extra API call.